packages/cli/test/simulation/multi_fork.test.ts

```diff
@@ -105,11 +107,41 @@ await checkpointSync.jobs.el.start();
 await checkpointSync.jobs.cl.start();
 await connectNewNode(checkpointSync.nodePair, env.nodes);
 
-await env.waitForNodeSync(rangeSync.nodePair);
-await env.waitForNodeSync(checkpointSync.nodePair);
-
+await waitForNodeSync(rangeSync.nodePair, env.options.controller.signal);
+await waitForNodeSync(checkpointSync.nodePair, env.options.controller.signal);
 await rangeSync.jobs.cl.stop();
 await rangeSync.jobs.el.stop();
 await checkpointSync.jobs.cl.stop();
 await checkpointSync.jobs.el.stop();
+
+// Unknown block sync
+// ========================================================
+const unknownBlockSync = env.createNodePair({
```

Comment:
For this node you must range sync

https://github.com/ChainSafe/lodestar/blob/30a88a2efcfff09145c7d0cb77bb877ad2b0317b/packages/beacon-node/test/e2e/sync/unknownBlockSync.test.ts#L83